splitwise/user.py

```diff
@@ -0,0 +1,19 @@
+import Splitwise
```

Comment:
You can use a more pythonic way of implementing getter and setter using the property method.
```python
    ...
    def __init__(self, name, email, mobile):
        self._name = name
    ...
    def __get_name(self):
        return self._name

    name = property(__get_name)
    ...
```